apps-devstg/us-east-1/k8s-kops/1-prerequisites/config.tf

```diff
@@ -0,0 +1,53 @@
+# Providers
+provider "aws" {
+  region  = var.canada_region_primary
```

Comment:
@juanmatias I'm wondering why `canada_region_primary` instead of just `region_primary` ?

Reply:
yeah, changed
I was the value I used for my deployment!